### ISSUE TITLE
P13 – App Transport Security

### ISSUE THREAD

**Issue by jessetipton** [MakeSchool-Tutorials/Makestagram-Swift-V2#19]
Parse seems to use HTTP (As opposed to HTTPS) requests to download `PFFile`s. We'll have to allow instructions to allow arbitrary loads in the tutorial

**Comment by danielhaaser** [comment]
I haven't run into this issue - is your parse server configured correctly to use https?

For example, my Parse server is located at: https://makestagram-parse-djh.herokuapp.com/ and it's https in my AppDelegate configuration also.

**Comment by jessetipton** [comment]
Yes my Parse server is configured correctly to use HTTPS. I asked around during the stand up and no one else has seen this issue 😕

**Comment by danielhaaser** [comment]
I'm beginning to think this might have to do with ACLs. I can retrieve images uploaded by the logged in user over HTTPS, but I get the app transport security issue when trying to retrieve images uploaded by different users. Is it the same for you?
